web/ui/react-app/src/ExpressionInput.tsx

```diff
@@ -72,9 +73,9 @@ class ExpressionInput extends Component<ExpressionInputProps> {
                   },
                 })}
               >
-                {/* TODO: Find better way than setting inner HTML dangerously. We just want the <strong> to not be escaped.
-                    This will be a problem when we save history and the user enters HTML into a query. */}
-                <span dangerouslySetInnerHTML={{__html: item.string}}></span>
+                <SanitizeHTML inline={true}>
```

Comment:
Could we explicitly only allow the `<strong>` tag?

Reply:
Yes, improved `SanitizeHTML` component to accept props `allowedTags` using which it's whitelisting tags.